ChurchCRM/CRM
# Improves the dev docker environment usability

#### What's this PR do?
Improves the dev docker environment usability


#### Screenshots (if appropriate)
N/A

#### What Issues does it Close?
N/A

Closes

#### What are the relevant tickets?
N/A

#### Any background context you want to provide?
Trying to make the docker dev/test environment a bit more flexible and usable.

#### Where should the reviewer start?
- In the `docker` directory

#### How should this be manually tested?
- Make sure the latest version of node and npm are installed in the webserver container
- Try logging in to the dev web server container using `npm run docker-dev-login-web`
- Try logging in to the dev database container using `npm run docker-dev-login-web`
- Ability to access adminer on port 8088 when using the docker dev environment

#### How should the automated tests treat this?
N/A

#### Questions:
- Is there a related website / article to substantiate / explain this change?
  -  [x ] No

- Does the development wiki need an update?
  -  [x ] No

- Does the user documentation wiki need an update?
  -  [ x] No

- Does this add new dependencies?
  -  [x ] No

- Does this need to add new data to the demo database
  -  [x ] No
